.env-template

```diff
@@ -0,0 +1,3 @@
+COOKIE_SECRET={{cookieSecret}}
+MONGO_URI={{mongoUri}}
+
```

Comment:
Add:
`CLOUDINARY_URL={{cloudinaryUrl}}`

Reply:
Right now the site is setup to use local file storage vs uploading the file to cloudinary. Do we want to setup a cloudinary account to upload images to?

Reply:
Removed the gallery model and pages as we are not using this functionality now or possibly in the future.